sip/sip-000-stacks-improvement-proposal-process.md

```diff
@@ -0,0 +1,111 @@
+
+**Obsolete**
+When a SIP is no longer relevant or is superseded by another SIP it’s status will change to Obsolete. 
+
+**Deferred**
+When no progress has been made on a SIP for a period of time, it can be marked as deferred.
+
+**Rejected**
+A SIP can be marked as rejected if the SIP is unable to gather enough community support.
+
+**Withdrawn** 
+The SIP author may choose to withdraw the SIP.
+
+## SIP format and structure
+
+The SIP should contain the following sections
+
+**Preamble header**
+A metadata header containing the following information:
+
+- Title
+- Author
+- Status
+- Type (Standard, Process or Informational)
+- Created Date
+- License
+- Comments URL (Link to discussion for this SIP)
+- Replaces/Superseded by (Optional)
+
+**Abstract**
+A short (~200 word) description of the issue being addressed.
+
+**Specification**
+The detailed description of the new feature or process.
+
+**Rationale**
+The rationale and motivation behind the new feature or process. It should describe any alternate designs and how the decision was made.
+
+**Reference implementation**
+Standards Track SIPs consist of two parts, a design document and a reference implementation. The SIP should be reviewed and accepted before a reference implementation is begun, unless a reference implementation will aid people in studying the SIP. Standards Track SIPs must include an implementation -- in the form of code, a patch, or a URL to same -- before it can be considered Final.
+
+## SIP Types
+
+**Standard**
+A SIP of this type describes a new feature or improvement to the Stacks blockchain protocol that affects all Stacks blockchain implementations.
+
+**Process**
+A process SIP is a document describing a new process surrounding the Stacks blockchain.
+
+**Informational**
+An informational SIP describes a design issue or provides general guidelines. Users and implementors are free to ignore informational SIPs.
+
+## SIP Comments
+
+It is the responsibility of the author to circulate the SIP and gather input from the community. Comments on the SIP can be added directly to the GitHub pull request or on a SIP-specific discussion thread on the [Blockstack forums](https://forum.blockstack.org). In the latter case, the link to the forum thread should be indicated in the preamble header.
+
+## Core Developer Committee
+
+The core developer committee is a group of active contributors to the Stacks blockchain. They are tasked with reviewing SIPs and managing their progression. Initially the committee will consist of developers from Blockstack PBC. Membership will be opened to the community once mining begins. Nominations to join the committee may be submitted by an existing committee member and approval requires a majority of committee members.
+
+## Copyright & Licensing
+
+Each SIP must identify at least one acceptable license in its preamble. Source code in the SIP can be licensed differently than the text. Below is a list of recommended licenses.
```

Comment:
We might want to clarify that we expect an OSI-sanctioned license, and mention that the list here is not exhaustive.  For example, the Stacks blockchain reference implementation is GPLv3.

Reply:
Shouldn't the license simply always be GPLv3 for contributions to blockstack-core ?

Reply:
Yeah, I'm fine with that.  We could simply state that SIPs whose reference implementation(s) touch existing reference implementation(s) must use the same license as the existing implementation(s) in order to be considered.